xsec/crossSectionDataFromFile.C

```diff
@@ -188,16 +188,16 @@ void ScaleBG(Variable* var, CCPi::MacroUtil& util, const CVHW& loW_wgt,
 // Main
```

Comment:
These types of changes don't need to be committed.

Smaller branches/pull requests are better.